src/enclave/outside/ocalls.cpp

```diff
@@ -75,19 +75,14 @@ extern "C"
     int32_t ocallSbrk(int32_t increment)
     {
         SPDLOG_TRACE("S - __sbrk - {}", increment);
-        wasm::WasmModule* module = wasm::getExecutingModule();
-        uint32_t oldBrk = module->getCurrentBrk();
-
-        if (increment == 0) {
-            return oldBrk;
-        } else if (increment < 0) {
-            module->shrinkMemory(-1 * increment);
-            return oldBrk;
-        } else {
-            return module->growMemory(increment);
-        }
+        SPDLOG_WARN("SGX-WAMR sbrk does not allocate more memory");
```

Comment:
We don't really want to be growing memory outside the enclave. Memory management will have to be implemented inside the enclave.

Reply:
Should this be an error then?

Reply:
We still call this method from SGX code, so even if we do nothing we must not error out in order for functions to finish.